src/mongoClusters/tree/MongoClusterResourceItem.ts

```diff
@@ -57,18 +55,11 @@ export class MongoClusterResourceItem extends MongoClusterItemBase {
 
                 const clusterConnectionString = nonNullValue(clusterInformation.connectionString);
 
-                context.valuesToMask.push(clusterConnectionString);
-
-                // Fetch non-admin users using the extracted method
-                const clusterNonAdminUsers = await this.fetchNonAdminUsersFromAzure(
-                    managementClient,
-                    clusterInformation,
-                );
+                context.valuesToMask.push(clusterConnectionString, clusterInformation.administratorLogin ?? '');
```

Comment:
perf nit: we should not push empty values, it might same some ticks here, but telemetry call will need to reiterate on that empty string. I know in this case it's a noop, but let's have a separate 
```
if (clusterInformation.administratorLogin) {
    context.valuesToMask.push(clusterInformation.administratorLogin);
}
```
just to keep awareness of performance since in other cases it might have a bigger impact.

Reply:
you're right, I remember typing that quick `?? ''` and knowing that I do not like it at all.. I'll pay more attention to it. Thanks!